cli/npm/registry.rs

```diff
@@ -293,6 +297,16 @@ impl NpmRegistryApi {
       );
     }
 
+    if self.no_npm {
```

Comment:
I think we should remove this? I don't think it's necessary because the `add_package_reqs` will take care of everything.

Reply:
That means we'll still be hitting registry even with `--no-npm` flag.

Reply:
We won't be because the only way that will happen is if there are npm package requirements.

Reply:
Sounds good, I'll do that!